modules/ROOT/pages/sandboxes.adoc

```diff
@@ -0,0 +1,39 @@
+= Sandboxes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+After you install an Anypoint API Community Manager update, it is not possible to undo the installation
+and return to a previous version. Follow the best-practice workflow by creating safe sandbox
+environments and using them to test each update before installing the update in your production environment.
+
+Best-practice workflow:
```

Comment:
Likewise, how about a sentence, something like "Follow this workflow to test an update before using it:"

Reply:
Actually...this is a task, not a workflow (imperative voice). You might want to reformat this as a topic set:

= Create and Use Sandboxes as Safe Testing Environments
intro
== Sandbox Testing Workflow (although these are really use steps and probably should have a task title...which is complicated by the following subtopic being...:
== Create a Sandbox